web/impact/impact/middleware/method_override_middleware.py

```diff
@@ -0,0 +1,13 @@
+# MIT License
+# Copyright (c) 2017 MassChallenge, Inc.
+
+METHOD_OVERRIDE_HEADER = 'HTTP_X_HTTP_METHOD_OVERRIDE'
+
+
+class MethodOverrideMiddleware(object):
+    def process_request(self, request):
+        if request.method != 'POST':
+            return
+        if METHOD_OVERRIDE_HEADER not in request.META:
+            return
+        request.method = request.META[METHOD_OVERRIDE_HEADER]
```

Comment:
I think changewise this all looks good - I think it might be adding a mention of this functionality in the README or quickstart document about how this works.